### PR TITLE
Make HLTDisplacedmumumuVtxProducer a stream producer

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
#### PR description:

Make HLTDisplacedmumumuVtxProducer a stream producer
Based on CMSSW_11_0_0_pre10

#### PR validation:

TSG tests

#### if this PR is a backport please specify the original PR:

N/A

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Review by Dr15Jones — CHANGES_REQUESTED** [review]

**Comment by Dr15Jones** [comment]
Did I miss something? On a quick look at the code it appears this could be a global module instead.

File: HLTrigger/btau/plugins/HLTDisplacedmumumuVtxProducer.h [hunk 1]
```diff
@@ -13,41 +13,42 @@
...
 }
 
-class HLTDisplacedmumumuVtxProducer : public edm::EDProducer {
+class HLTDisplacedmumumuVtxProducer : public edm::stream::EDProducer<> {
 public:
   explicit HLTDisplacedmumumuVtxProducer(const edm::ParameterSet&);
   ~HLTDisplacedmumumuVtxProducer() override;
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
-  void beginJob() override;
+  virtual void beginJob();
```
[the remark this answers is not in the document]
Comment (Martin-Grunewald) [reply HLTrigger/btau/plugins/HLTDisplacedmumumuVtxProducer.h]: I guess I can just remove them, given they are empty in the cc files...

File: HLTrigger/btau/plugins/HLTDisplacedmumumuVtxProducer.h [hunk 2]
```diff
@@ -13,41 +13,42 @@
...
   void produce(edm::Event&, const edm::EventSetup&) override;
-  void endJob() override;
+  virtual void endJob();
```
[the remark this answers is not in the document]
Comment (Martin-Grunewald) [reply HLTrigger/btau/plugins/HLTDisplacedmumumuVtxProducer.h]: I guess I can just remove them, given they are empty in the cc files...

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by fabiocos** [comment]
@Martin-Grunewald any comment to https://github.com/cms-sw/cmssw/pull/28175#issuecomment-542729127 ?

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by fabiocos** [comment]
+1

@Martin-Grunewald thanks for addressing the comment of @Dr15Jones

**Pull request merged**